### DISCUSSION TITLE
% does not match actual speed

### DISCUSSION THREAD

**Discussion by empty1958** [Rem0o/FanControl.Releases#793] [Q&A] [answered]
I've noticed on my system other than 100% the fan speed % shown does not match the actual fan speed. An example, to get the fan to run at 50% of it's max I set a flat curve at 38%. Is there an adjustment/setting that can correct this? Or am I just missing something?

Thank You

**Comment by Rem0o** [comment, marked as answer]
% is actually % of "power" provided to the fan.

Relative to the speed (RPM), it will be non-linear.

**Reply by empty1958** [reply]
Thanks for the quick reply
